batch/plugins/org.jboss.tools.batch.core/src/org/jboss/tools/batch/core/BatchCorePlugin.java

```diff
@@ -28,6 +34,23 @@ public static BatchCorePlugin getDefault() {
 		return plugin;
 	}
 
+	@Override
+	public void start(BundleContext context) throws Exception {
```

Comment:
shouldn't you remove it on stop()?
if you stop the plugin via the osgi console and restart it, wouldn't it register 2 ISaveParticipants? ( no idea if there's a built-in deduplication mechanism in addSaveParticipant.